Dockerfile.3.5

```diff
@@ -18,7 +18,12 @@ ENV \
 	READY_FILE=/run/ready \
 	TEMPLATE_DIR=/etc/deploy/templates \
 	WEBPACK_CONFIG=/app/docker/webpack.config.js \
-	PACKAGE_JSON=/app/docker/package.json
+	PACKAGE_JSON=/app/docker/package.json \
+	PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LANG=C.UTF-8 \
+	PORT=8080 \
+	HOST=0.0.0.0
```

Comment:
I think these should get exposed via build arguments.  https://docs.docker.com/engine/reference/builder/#arg